### PR TITLE
Automatic Release-Action for OEE-Simulator

### PR DESCRIPTION
**Pull request by pejo-sag**

### CODE REVIEW

File: .github/workflows/build-release.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          asset_content_type: application/zip
+
+      - name: upload source-code zip
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: 'source-code.zip'
+          asset_name: 'Source Code (zip)'
```
Comment (mvoigt-sag) [inline .github/workflows/build-release.yml:85]: maybe name this 'oee-simulators-sources.zip' oder 'oee-simulators-src.zip'?

File: .github/workflows/build-release.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    inputs:
+        release_body:
+          type: string
+          description: GitHub Release Message
+          default: "GitHub Release Message"
+  push: 
+    branches: [ COE-722 ]
+
+env:
+  BUILD_OFFSET: 25
```
Comment (mvoigt-sag) [inline .github/workflows/build-release.yml:16]: maybe we run the workflow 15 more time, then the build number will be higher than 30 (and therefore higher than the last release) and we can get rid of this ;-)

File: .github/workflows/build-release.yml (hunk 1 above)
Comment (pejo-sag) [reply .github/workflows/build-release.yml]: Done

File: .github/workflows/build-release.yml (hunk 2 above)
Comment (pejo-sag) [reply .github/workflows/build-release.yml:14]: Seems reasonable 🦖

**Review by mvoigt-sag — APPROVED** [review]

**Pull request merged**